__tests__/integration/schema/core.js

```diff
@@ -1,6 +1,17 @@
 const printSchemaOrdered = require("../../printSchemaOrdered");
 const { withPgClient } = require("../../helpers");
 const { createPostGraphileSchema } = require("postgraphile-core");
+const { isSchema } = require("graphql/type/schema")
```

Comment:
This is also exported direct from `graphql`, please use that instead:

```suggestion
const { isSchema } = require("graphql")
```